.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+# Runs manually
+on:
+  workflow_dispatch:
+      
+jobs:
+  release:
+    name: Release on Sonatype OSS
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Apache Maven Central
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'temurin'
+          cache: 'maven'
+          server-id: ossrh # Value of the distributionManagement/repository/id field of the pom.xml
+          server-username: SONATYPE_USERNAME # env variable for username in deploy
+          server-password: SONATYPE_PASSWORD # env variable for password in deploy
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: GPG_PASSPHRASE # env variable for GPG private key passphrase
+
+      - name: Publish to Apache Maven Central
```

Comment:
before doing this, you need to remove the `-SNAPSHOT` from the version. See here:
https://github.com/devonfw/ide/blob/964898a0189a964eadf01c2dbbdc2b2806c19702/scripts/src/main/resources/scripts/command/release#L53
https://github.com/devonfw/ide/blob/964898a0189a964eadf01c2dbbdc2b2806c19702/scripts/src/main/resources/scripts/command/release#L63
https://github.com/devonfw/ide/blob/964898a0189a964eadf01c2dbbdc2b2806c19702/scripts/src/main/resources/scripts/command/release#L102
https://github.com/devonfw/ide/blob/964898a0189a964eadf01c2dbbdc2b2806c19702/scripts/src/main/resources/scripts/command/release#L103
https://github.com/devonfw/ide/blob/964898a0189a964eadf01c2dbbdc2b2806c19702/scripts/src/main/resources/scripts/command/release#L104

Please note that `devon build set-version/get-version` does not have to be re-implemented here with all its huge complexity. We know that we have `.mvn/maven-config` file here, so you can do that directly:
So you can find the magic here from this line on:
https://github.com/devonfw/ide/blob/964898a0189a964eadf01c2dbbdc2b2806c19702/scripts/src/main/resources/scripts/command/mvn#L89

Reply:
OK, I meanwhile saw that you have figured out all of this already but it is below in the `gitHub_release` job.
But doesn't run github action runner the jobs in their order of declaration or am I missing some understanding of github actions here?

Reply:
yes that's right. I'll rework that.